src/lib.rs

```diff
@@ -1869,7 +1866,7 @@ impl From<wasi_filesystem::DescriptorType> for wasi::Filetype {
 
 // A black box to prevent the optimizer from generating a lookup table
 // from the match above, which would require a static initializer.
-fn black_box(x: Errno) -> Errno {
+fn obscure(x: Errno) -> Errno {
```

Comment:
Could we use https://doc.rust-lang.org/std/hint/fn.black_box.html now that it's stable?

Reply:
Cool, yes, that's better.